### DISCUSSION TITLE
upgrade to angular 12 and bootstrap 5

### DISCUSSION THREAD

**Discussion by yahyaKacem** [NG-ZORRO/ng-zorro-antd#6704] [General] [answered]
Will this be upgraded to angular 12 and bootstrap 5?
Or is it compatible with both?

**Comment by vthinkxie** [comment, marked as answer]
Hi, ng-zorro-antd follow the spec with ant design, there is nothing todo with bootstrap

we will support angular 12 in next month.

**Reply by seytzhan** [reply]
Which next month do you mean ?
